pkg/importer/writer.go

```diff
@@ -210,8 +243,12 @@ func buildPathString(path string, params []Param) string {
 }
 
 func (w *writer) writeEndpoint(method string, endpoint Endpoint) {
+	responseContentType := ""
```

Comment:
nit: try using `var responseContentType string` when declaring but not initialising

Reply:
Addressed